packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -276,9 +273,12 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         if (!this.channelDeferred) {
             this.channelDeferred = new Deferred<IFluidDataStoreChannel>();
             this.realizeCore(this.existing).catch((error) => {
-                this.channelDeferred?.reject(
-                    CreateProcessingError(error, "realizeFluidDataStoreContext", undefined /* message */));
-            });
+                const errorWrapped = CreateProcessingError(error, "realizeFluidDataStoreContext");
+                this.channelDeferred?.reject(errorWrapped);
+                this.logger.sendErrorEvent(
+                    { eventName: "RealizeError", fluidDataStoreId: { value: this.id, tag: "PackageData"} },
```

Comment:
Why not put `fluidDataStoreId` prop on the error used for `reject` via `addTelemetryProps`?